### PR TITLE
New Formula: xhyve

### PR DESCRIPTION
**Pull request by prefork**
xhyve: a lightweight OS X virtualization solution
https://github.com/mist64/xhyve

### CODE REVIEW

File: xhyve.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  desc "xhyve is a port of bhyve to OS X"
+  homepage "https://github.com/mist64/xhyve"
+  head "https://github.com/mist64/xhyve.git"
+
+  depends_on MinimumMacOSRequirement => :yosemite
+
+  def install
+    system "make"
+    bin.install "build/xhyve"
+  end
```
Comment (bfontaine) [inline xhyve.rb]: Could you add a test, using a `test do` block? Thanks!

File: xhyve.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+  test do
+    system "xhyve -h"
```
Comment (denji) [inline xhyve.rb]: `system "xhyve", "-h"`

**Comment by prefork** [comment]
Thanks for all your help so far.  This failed on Mavericks and Mountain Lion because of the `MinimumMacOSRequirement` clause but passed on Yosemite.  What's next?

**Comment by denji** [comment]
@mikemcquaid I wonder what travis calls a fatal failed when calling the minimum supported platform [`minimum_macos_requirement.rb#L4`](https://github.com/Homebrew/homebrew/blob/5bdc5991/Library/Homebrew/requirements/minimum_macos_requirement.rb#L4)

File: xhyve.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+class Xhyve < Formula
+  desc "xhyve is a port of bhyve to OS X"
+  homepage "https://github.com/mist64/xhyve"
+  head "https://github.com/mist64/xhyve.git"
+
+  depends_on MinimumMacOSRequirement => :yosemite
+
+  def install
+    system "make"
+    bin.install "build/xhyve"
```
Comment (mikemcquaid) [inline xhyve.rb]: Is there no `make install` we can use?

File: xhyve.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  end
+
+  test do
+    system "xhyve", "-h"
```
Comment (mikemcquaid) [inline xhyve.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: xhyve.rb (hunk 3 above)
Comment (prefork) [inline xhyve.rb]: @mikemcquaid Not in the [current version](https://github.com/mist64/xhyve/blob/84868be7a65305a3a7ab8f3c197d6df93ebbe5dc/Makefile).

File: xhyve.rb (hunk 4 above)
Comment (prefork) [inline xhyve.rb]: @mikemcquaid I'll have to think about this a bit more.  The [only other thing](https://github.com/mist64/xhyve/blob/84868be7a65305a3a7ab8f3c197d6df93ebbe5dc/src/xhyve.c#L842) the application can do is spawn a vm, which might be tough depending on how the CI environment is configured.

Is the yosemite build slave physical or virtual?

Comment (mikemcquaid) [inline xhyve.rb]: Virtual. Sounds find to leave as-is, thanks!

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for new formulae is `xhyve (new formula)`.

**Pull request closed without merging**